packages/core/src/components/XMarkdownCore/hooks/useMarkdown.ts

```
@@ -0,0 +1,42 @@
+import { flow } from 'lodash-es';
+
+export function useProcessMarkdown(markdown: string) {
+  return preprocessLaTeX(markdown);
+}
+
+export function preprocessLaTeX(markdown: string) {
+  if (typeof markdown !== 'string') return markdown;
```

ESLint

[error] Expect newline after if

(antfu/if-newline)